sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -515,15 +537,25 @@ func (m *networkFlowManager) enrichConnection(conn *connection, status *connStat
 		}
 
 		if extSrc == nil {
+			entityType := networkgraph.InternetEntity()
+			isExternal, err := conn.IsExternal()
+			if err != nil { // IP is malformed or unknown - assume it is external and log a warning
+				log.Warnf("Not showing flow on the network graph: %v", err)
```

Comment:
Question to reviewers: On OCP there is one occurrence of that warning every 30s. Apparently, Collector reports an empty IP, or Sensor somehow looses the data. Shall this be a warning, or shall we change it to debug?

Reply:
Here is the ticket for Collector bug (including reproduction steps): https://issues.redhat.com/browse/ROX-22388

I will add a comment in the code to reference that ticket.